lib/matplotlib/pyplot.py

```diff
@@ -575,6 +575,7 @@ def gcf():
 
 
 def fignum_exists(num):
+    """Return whether the figure with the given number exists."""
```

Comment:
Technically figure numbers don't need to be numbers; I'm not sure how to convey that succinctly in the one line though.

Reply:
We could use `id` instead of `num`. Would it be permissible to change the argument name as well? Technically, that would be a breaking change if someone used it as a kwarg. However, thats rather unlikely, as there is no good reason to do it here?